File: ejercicio6tp1/ejercicio11.java
import java.util.Random;
import java.util.Scanner;

public class ejercicio11 {
    
    public static void main(String[] args){
        Random random = new Random();
        Scanner scanner = new Scanner(System.in);
        boolean rta = true;
        System.out.print("Numeros aleatorios.");
        while (rta) {
            int num1 = random.nextInt(10);
            int num2 = random.nextInt(10);
            while (num1==num2) {
                num1 = random.nextInt(10);
                num2 = random.nextInt(10);
            }
            System.out.println("Desea apostar por el valor 1: " + num1 + " o por el valor 2: " + num2 + " ?");
            int resp = scanner.nextInt();
            while (resp != 1 && resp != 2){
                System.out.print("Error, ingrese una respuesta correcta! (1/2)");
                resp = scanner.nextInt();
            }
            if (num1>num2){
                if (resp==1) System.out.print("Has ganado!");
                else System.out.print("Has perdido!");
                }
            else {if (resp==1) System.out.print("Has perdido!");
                else System.out.print("Has ganado!");
            }
            System.out.print("Desea jugar otra vez? s/n");
            scanner.nextLine();
            String respuesta = scanner.nextLine();
            if (respuesta != "s" && respuesta != "n") {
                System.out.print("Error, ingrese una respuesta correcta! (s/n)");
                respuesta = scanner.nextLine();
            }
            rta = respuesta.equalsIgnoreCase("s");
        }
    scanner.close();
    }
}

File: TP1/TP1 python/python fede/ejercicio11.py
import random

rta = True
num1 = random.randint(1,10)
num2 = random.randint(1,10)

while rta==True:
    while num1==num2:
        num1 = random.randint(1,10)
        num2 = random.randint(1,10)
    print("Valor 1:", num1)
    print("Valor 2:", num2)
    ap = int(input("Desea apostar por el valor 1 o por el valor 2?: "))
    while ap != 1 and ap != 2:
        ap = int(input("Error! Ingrese un valor correcto (1/2):"))
    if (ap==1):
        if (num1>num2):
            print("Has ganado!")
        else:
            print("Has perdido!")
    else:
        if (num1>num2):
            print("Has perdido!")
        else:
            print("Has ganado!")
    resp = input("Desea jugar otra vez? (s/n): ")
    while (resp != "s") and (resp != "n"):
        resp = input("Error! Ingrese una respuesta correcta (s/n): ")
    if resp == "s":
        rta = True
    else:
        rta = False
print("Gracias por jugar!")